dom.bs

```diff
@@ -958,48 +958,48 @@ are not to be used for anything else. [[!HTML]]
 
 <dl class=domintro>
  <dt><code><var>target</var> = new <a constructor for=EventTarget lt=EventTarget()>EventTarget</a>();</code>
- <dd>
-  Creates a new {{EventTarget}} object, which can be used by developers to <a>dispatch</a> and listen for
-  <a>events</a>.
+ <dd><p>Creates a new {{EventTarget}} object, which can be used by developers to <a>dispatch</a> and
+ listen for <a>events</a>.
 
  <dt><code><var>target</var> . <a method for=EventTarget lt=addEventListener()>addEventListener</a>(<var>type</var>, <var>callback</var> [, <var>options</var>])</code>
  <dd>
-  Appends an <a>event listener</a> for <a>events</a> whose {{Event/type}} attribute value
-  is <var>type</var>. The <var>callback</var> argument sets the <b>callback</b> that will
-  be invoked when the <a>event</a> is <a>dispatched</a>.
+  <p>Appends an <a>event listener</a> for <a>events</a> whose {{Event/type}} attribute value is
+  <var>type</var>. The <var>callback</var> argument sets the <a for="event listener">callback</a>
+  that will be invoked when the <a>event</a> is <a>dispatched</a>.
 
-  The <var>options</var> argument sets listener-specific options. For compatibility this can be just
-  a boolean, in which case the method behaves exactly as if the value was specified as
+  <p>The <var>options</var> argument sets listener-specific options. For compatibility this can be a
+  boolean, in which case the method behaves exactly as if the value was specified as
   <var>options</var>' <code>capture</code> member.
 
-  When set to true, <var>options</var>' <code>capture</code> member prevents <b>callback</b> from
-  being invoked when the <a>event</a>'s {{Event/eventPhase}} attribute value is
-  {{Event/BUBBLING_PHASE}}. When false (or not present), <b>callback</b> will not be invoked when
-  <a>event</a>'s {{Event/eventPhase}} attribute value is {{Event/CAPTURING_PHASE}}. Either way,
-  <b>callback</b> will be invoked if <a>event</a>'s {{Event/eventPhase}} attribute value is
-  {{Event/AT_TARGET}}.
+  <p>When set to true, <var>options</var>' <code>capture</code> member prevents
```

Comment:
While here you could consistify to use "options's" per https://wiki.whatwg.org/wiki/Style#Grammar

Reply:
I thought there was still an exception when the word is plural?

Reply:
We've had this discussion before :). `options` is not a class of values; it is a single JS value. So it's not really plural in the grammatical sense that would trigger leaving off the s, where `capture` belongs to a collection of options.

Stated another way, here "options" is short for "options variable", which is not plural.

A case where it would be plural is "the stock options' combined value was a billion dollars."